care_o_bot_robot/package.xml

```diff
@@ -13,7 +13,13 @@
   <exec_depend>cob_bringup</exec_depend>
   <exec_depend>cob_manipulation</exec_depend>
   <exec_depend>cob_navigation</exec_depend>
+  <exec_depend>htop</exec_depend>
+  <exec_depend>openssh-server</exec_depend>
+  <exec_depend>python-wstool</exec_depend>
   <exec_depend>robot_upstart</exec_depend>
+  <exec_depend>tmux</exec_depend>
+  <exec_depend>tree</exec_depend>
+  <exec_depend>vim</exec_depend>
```

Comment:
Ok then, I approve to add them here...
still please consider https://github.com/ipa320/care-o-bot/pull/38#issuecomment-303103713